README.md

```diff
@@ -2,6 +2,8 @@
 
 ![build](https://github.com/bcgsc/pori_graphkb_loader/workflows/build/badge.svg?branch=master) ![Docker Image Version (latest semver)](https://img.shields.io/docker/v/bcgsc/pori-graphkb-loader?label=docker%20image) ![node versions](https://img.shields.io/badge/node-10%20%7C%2012%20%7C%2014-blue)
 
+This repository is part of the [platform for oncogenomic reporting and interpretation](https://github.com/bcgsc/pori).
+
```

Comment:
Are you going with 'the PORI' instead of 'PORI'?

Reply:
just when used in a sentence like this. If it were an acronym I would probably use just PORI, the PORI sounds odd